spec/ddtrace/contrib/rails/rails_auto_instrument_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'ddtrace/contrib/rails/rails_helper'
+
+RSpec.describe 'Datadog::AutoInstrument' do
+  include_context 'Rails test application'
+
+  around do |example|
+    # Reset before and after each example; don't allow global state to linger.
+    Datadog.configuration.reset!
+    example.run
+    Datadog.configuration.reset!
+  end
+
+  context 'when auto patching is included' do
+    before do
+      skip 'Java not supported' if RUBY_PLATFORM == 'java'
```

Comment:
```suggestion
      skip if PlatformHelpers.jruby?
```

I normally remove the `skip` message if the `if` condition is an exact copy of the conditional statement, but that's just a preference.

Reply:
makes sense, updated